canu/inventory/plays/dell-os10.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: sw-cdu-001
+  collections:
+    - dellemc.os10
+  gather_facts: true
+  vars:
+    # This should be a template but for now, pass is an extra var path to whatever config is needed
+    # pass in the folder containing the switch configs or be in the same directory
+    config_folder: ""
+    switch_config: "{{ config_folder }}/{{ inventory_hostname }}.cfg"
+
+  tasks:
+    - name: dry run
```

Comment:
I was going to make dryrun a role so it could be re-used.  dell does it different than aruba it looks like, but the role can come later if you're already on a move with the plays